packs/pathfinder-npc-core/ancestry-npcs/elf/forlorn-artist.json

```
@@ -30,7 +30,7 @@
                 "slots": {},
                 "slug": null,
                 "spelldc": {
-                    "dc": 0,
+                    "dc": 18,
                     "value": 0
```

```suggestion
                    "value": 10
```
The spell attack for NPCs is DC - 8 if not listed, which is necessary data to have for counteracting purposes.